src/pybind/mgr/dashboard/settings.py

```diff
@@ -57,6 +57,9 @@ class Options(object):
     ENABLE_BROWSABLE_API = Setting(True, [bool])
     REST_REQUESTS_TIMEOUT = Setting(45, [int])
 
+    # CREATE CLUSTER ENABLE/DISABLE
+    INITIALIZE_CLUSTER_CREATION = Setting(True, [bool])
```

Comment:
@epuertat It depends on the behavior: the user goes to the wizard and for some reason it does not complete the cluster creation the first time, are we going to redirect him always to cluster creation the next times or only the first time? If only the first time, then the suggestion is not appropriate because you don't know the final state.
As an example, in DDD naming is about the use case, not the final state, so if we don't go for ```INITIALIZE_CLUSTER_CREATION``` what about "CLUSTER_WIZARD_VISITED", which is more precise?

Reply:
@alfonsomthd IMHO the Wizard is just an implementation detail of the presentation layer (it might well be a form, a table, or any other UI pattern). What matters to the back-end is the configuration status of the cluster (final state = configured or created): how you do that (the transition from non-configured to configured, with a Wizard/Form/etc.) is IMHO a front-end concern.

Reply:
@epuertat
> the user goes to the wizard and for some reason it does not complete the cluster creation the first time, are we going to redirect him always to cluster creation the next times or only the first time?

We need to answer this question to know if we need 2 settings, 1 reflecting the current state (more precise than ```final```) and other to know if we should still show the wizard (as a sysadmin, maybe I don't want to be redirected to the wizard constantly if I prefer to do things manually).
For the sake of simplification, we can keep only 1 setting and show the wizard ```1 time if and only if the cluster is not completed```. Then, if for whatever reason the operation is not completed, the redirection will not happen again.

Reply:
> If a user wants to skip/dismiss the configuration wizard, from the perspective of the cluster it makes no difference: the cluster should be considered 'configured'.

No, if the user does not complete the configuration the cluster is not configured.

Reply:
As discussed with @alfonsomthd and @nizamial09, the agreement is to keep 1 statuses:
* `INSTALLED`
  * Status after `cephadm bootstrap`.
  * In this status, the wizard will always be displayed to the admin user (there shouldn't be other accounts, as the cluster is fresh).
  * This status only changes to `POST_INSTALLED` when the Wizard completes successfully or the user "skips" the Wizard.
* `POST_INSTALLED`
  * The Wizard is no longer displayed.
  * Shall we keep the wizard URL working or it might be harmful?.

![image](https://user-images.githubusercontent.com/37327689/118806339-3a2cdd00-b8a7-11eb-97ea-fdf493783ccc.png)

Reply:
Made the changes according to this. FYI, for this PR I am setting POST_INSTALLED=true when the user clicks the Create Cluster button. Its for this PR only. On the next iteration I'll change this according to the cluster creation status.